go/pkg/clickhouse/schema/001_key_verifications_raw_v2.sql

```diff
@@ -0,0 +1,66 @@
+CREATE TABLE key_verifications_raw_v2
+(
+-- the api request id, so we can correlate the verification with traces and logs
+  request_id String,
+
+  -- unix milli
+  time Int64 CODEC(Delta, LZ4),
+
+  workspace_id String,
+  key_space_id String,
+  -- Empty string if the key has no identity
+  identity_id String,
+  key_id String,
+
+  -- Right now this is a 3 character airport code, but when we move to aws,
+  -- this will be the region code such as `us-east-1`
+  region LowCardinality(String),
+
+  -- Examples:
+  -- - "VALID"
+  -- - "RATE_LIMITED"
+  -- - "EXPIRED"
+  -- - "DISABLED
+  outcome LowCardinality(String),
+
+  tags Array(String) DEFAULT [],
+
+  -- The number of credits spent on this verification
+  -- 0 means no credits were spent
+  spent_credits Int64,
+
+  -- Latency in milliseconds for this verification
+  latency Float64,
+
+  INDEX idx_request_id (request_id) TYPE bloom_filter GRANULARITY 1,
+  INDEX idx_identity_id (identity_id) TYPE bloom_filter GRANULARITY 1,
+  INDEX idx_key_id (key_id) TYPE bloom_filter GRANULARITY 1,
+  INDEX idx_tags (tags) TYPE bloom_filter GRANULARITY 1
+)
+ENGINE = MergeTree()
+PARTITION BY toYYYYMMDD(fromUnixTimestamp64Milli(time))
+ORDER BY (workspace_id, time, key_space_id, outcome)
+TTL toDateTime(fromUnixTimestamp64Milli(time)) + INTERVAL 1 MONTH DELETE
+SETTINGS non_replicated_deduplication_window = 10000
```

Comment:
wouldn't it be possible for hot keys that get verified quite often that we drop valid verifications? We are storing the key verification per millisecond so im not sure how likely this is but it could happen

Reply:
why would we drop valid verifications?

Reply:
don't mind me I thoughht deduplication window was on the order by key but its for the full row and we have a unique request_id in there so its fine